nservicebus/gateway/index.md

```diff
@@ -97,3 +97,12 @@ To disable retries in the gateway use the `DisableRetries` setting:
 snippet: GatewayDisableRetriesConfiguration
 
 When retries are disabled, any messages that fail to be sent to another site will be immediately routed to the configured error queue.
+
+
+### Version compatibility
+
+The Gateway component ensures only forward compatibility for one major version. So a message sent by the NServiceBus 3.x Gateway can be understood by the NServiceBus 4.x Gateway, and a message sent by the NServiceBus 4.x Gateway can be understood by NServiceBus.Gateway 1.x (the independent package targeting NServiceBus 5.x).
```

Comment:
Next paragraph talks about (lack of) backwards compat. Perhaps too indirectly?

For instance when the V4 Gateway sends a message, it's going to be using the new SendSingleRequest type that the V3 Gateway knows nothing about. V3 would also be expecting an Ack that would never come.